### ISSUE TITLE
Project type not copied when duplicating a project

### ISSUE THREAD

**Issue by tobru** [OCA/project#1212]
## Module

`project_type`

## Describe the bug

When duplicating a project which has a project type configured, this is not copied over to the new project.
This is due to having `copy=False` in [project_project.py#L12](https://github.com/OCA/project/blob/16.0/project_type/models/project_project.py#L12).

## To Reproduce

**Affected versions**: `16.0.1.0.0`

**Expected behavior**

The project type is copied as well when duplicating a project.

**Comment by tobru** [comment]
I would be willing to contribute a fix with a PR, but would first like to have feedback from the community if this was feasible to change.

**Comment by francescapenso** [comment]
Also i've found this issue when duplicating and also when you create a new project from a template

**Comment by CRogos** [comment]
I also agree and support this. Would be helpful when using projects as a template in Projects:
![image](https://github.com/user-attachments/assets/f59eccf6-656e-4432-83fd-de6accd8332a)
